### ISSUE TITLE
Billing:- Billing Page Not Accessible via Keyboard Navigation

### ISSUE THREAD

**Issue by shouzab123** [criteriontech786/demo.Medvantage_Issues#152]
**Description:-**

The user is unable to control or navigate the page using the keyboard (e.g., Tab, Enter, Arrow keys).
Interactive elements such as buttons, links, dropdowns, or input fields are either not reachable or not operable via keyboard.

Step to Reproduce:-
1.	Open the web application page 
2.	Use Tab key to navigate through elements 
3.	Try to interact using Enter/Arrow keys

Actual Result:
User cannot navigate properly using keyboard

Expected Result:
All interactive elements should be accessible via keyboard

**Comment by MohdAdilHasan** [comment]
@mohdfurkan53  please tag @kamaljeet10193  for this issue

**Comment by kamaljeet10193** [comment]
It is already working, no issue to accessible via tab key.
